ui/app/rif/actions.js

```diff
@@ -193,14 +185,44 @@ function showMenu (data) {
   }
 }
 
+function navigateBack () {
```

Comment:
This was so necessary! Nice implementation

Reply:
Thanks man!